src/fft/polynomial.rs

```diff
@@ -109,7 +109,7 @@ impl Polynomial {
             .map(|(c, p)| p * c)
             .collect();
         let mut sum = BlsScalar::zero();
-        for eval in p_evals.into_iter() {
+        for eval in p_evals {
```

Comment:
Maybe we also change to Rust 2021?

Reply:
We should change to the `2021` edition once it its stable, to be on the safe side. I believe it's planned by the end of the month.

In addition, there are still few things in plonkup that are not following the `2018` edition, so we should ensure that first.